### PR TITLE
Roblox.RobloxStudio version 1.6.0.29627

### PR DESCRIPTION
**Pull request by markandpreston1**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/39866)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You forgot the `Publisher` change: https://github.com/microsoft/winget-pkgs/pull/39670/files#r777416699

**Comment by markandpreston1** [comment]
> You forgot the `Publisher` change: https://github.com/microsoft/winget-pkgs/pull/39670/files#r777416699

fixed @ItzLevvie

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

File: manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Roblox.RobloxStudio
+PackageVersion: 1.6.0.29627
```
Comment (ItzLevvie) [inline manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.installer.yaml:4]:
```suggestion
PackageVersion: 1.6.0.425477
```

File: manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Roblox.RobloxStudio
+PackageVersion: 1.6.0.29627
```
Comment (ItzLevvie) [inline manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 1.6.0.425477
```

File: manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Roblox.RobloxStudio
+PackageVersion: 1.6.0.29627
```
Comment (ItzLevvie) [inline manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.yaml:4]:
```suggestion
PackageVersion: 1.6.0.425477
```

File: manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Roblox.RobloxStudio
+PackageVersion: 1.6.0.29627
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://setup.rbxcdn.com/RobloxStudioLauncherBeta.exe
+  InstallerSha256: FF62991C65F22AC14AA9C53AB70D2D720A3C8A628E71CB305F46478310C513FD
```
Comment (ItzLevvie) [inline manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.installer.yaml:9]:
```suggestion
  InstallerSha256: AB515EF53C06C172329CB0DCAF75DB7291F62BCED759D6D24CE2DF1F186DBE0B
```

File: manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Roblox.RobloxStudio
+PackageVersion: 1.6.0.29627
+PackageLocale: en-US
+Publisher: Roblox | Published to database by Markandpreston
```
Comment (ItzLevvie) [inline manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.locale.en-US.yaml:6]:
```suggestion
Publisher: Roblox Corporation
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**